public/data/shahid.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Shahid Hussain",
+  "type": "personal",
+  "bio": "Aspiring Full-Stack Developer & Open Source Enthusiast!",
+  "avatar": "https://github.com/shahidhussain07.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shahidhussain07",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/shahidHtwts",
+      "icon": "twitter"
+    }
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/ig_shahidhussain",
+      "icon": "instagram"
+    }
```

Comment:
A missing comma here as well.
```suggestion
    },
```

Reply:
Thanks for addressing <3

Reply:
I have updated my JSON file as you've suggested.